homework/homework-05/readfile/filereader.go

```diff
@@ -0,0 +1,38 @@
+/*Not sure what shoulbe changed in example from the book. btw I foud and example
+where buffer is used. It looks more powerfull for files with huge amount of data*/
```

Comment:
In that case it's better to use `bufio.Reader` from `bufio` lib
just like that:
```
package main
import (
	"fmt"
	"os"
	"bufio"
	"io"
)
func main(){
	file, err := os.Open("some.data") 
	if err != nil { 
         fmt.Println("Unable to open file:", err) 
         return 
	} 
	defer file.Close() 
 
	reader := bufio.NewReader(file) 
	for { 
		line, err := reader.ReadString('\n') 
		if err != nil { 
			if err == io.EOF { 
				break 
			} else { 
				fmt.Println(err) 
				return 
			} 
		} 
		fmt.Print(line) 
	}
}
```